tools/utils.py

```diff
@@ -128,3 +128,6 @@ def is_msys() -> bool:
 
 def is_macos():
     return any(platform.mac_ver()[0])
+
+def split_release_version(version: str):
```

Comment:
Nit: `split_version_release` would be clearer because that's the order of the returned list.

Reply:
If I follow your logic, `split_version_revision` should be a little bit better, no?

Reply:
Oops, yes.